docs/options.md

```diff
@@ -142,19 +142,19 @@ which means it can be given multiple times.
 ```toml
 [tool.cibuildwheel]
 # Normal options, etc.
-manylinux-x86_64-image = "manylinux2010"
+manylinux-x86_64-image = "manylinux2014"
 
 [[tool.cibuildwheel.overrides]]
 select = "cp36-*"
 manylinux-x86_64-image = "manylinux1"
 
 [[tool.cibuildwheel.overrides]]
-select = "cp310-*"
+select = "cp3?-*"
 manylinux-x86_64-image = "manylinux2014"
 ```
 
-This example will build CPython 3.6 wheels on manylinux1, CPython 3.7-3.9
-images on manylinux2010, and CPython 3.10 wheels on manylinux2014.
+This example will build CPython 3.6 wheels on manylinux1, and manylinux2014
+wheels everywhere else.
```

Comment:
I admit I am confused by this example. Which stanza has priority? If the `select = "cp36-*"` overrides the default, then won't the `select = "cp3?-*"` re-override that to reset `cp36-*` to `manylinux2014`?

Reply:
Yes, you are right, that's exactly what it would do. The order needs to be swapped, or better yet, it could be `cp3{7,8,9}-*`. :/ I can do that if you want.

Reply:
Yes please